v20.1/admin-ui-overview-dashboard.md

```diff
@@ -48,19 +48,24 @@ The timeseries data used to power the graphs in the Admin UI is stored within th
 
 <img src="{{ 'images/v20.1/admin_ui_capacity.png' | relative_url }}" alt="CockroachDB Admin UI Capacity graph" style="border:1px solid #eee;max-width:100%" />
 
-You can monitor the **Capacity** graph to determine when additional storage is needed.
+You can monitor the **Capacity** graph to determine when additional storage is needed (e.g., by scaling your cluster). 
 
-- In the node view, the graph shows the maximum allocated capacity, available storage capacity, and capacity used by CockroachDB for the selected node.
+Metric | Description
+--------|--------
+**Capacity** | The maximum store size. This value may be set per node using [`--store`](cockroach-start.html#store). If a store size has not been set, this metric displays the actual disk capacity. See [Capacity metrics](#capacity-metrics).
+**Available** | The free disk space available to CockroachDB data.
+**Used** | The disk space in use by CockroachDB data. This excludes the Cockroach binary, operating system, and other system files.
```

Comment:
This seems to be related to the sentence about disk space I was confused by earlier.  Maybe this phrasing is clearer, and could be re-used there.